io.embrace.sdk/Editor/EmbracePostBuildProcessor.cs

```diff
@@ -238,55 +239,28 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
             var resourcesFilesGuid = project.AddFile(EmbracePlistName, "/" + EmbracePlistName, PBXSourceTree.Source);
             project.AddFileToBuildSection(appTargetGuid, resourcesBuildPhase, resourcesFilesGuid);
             
-            // Embed Embrace.framework
-
-            string xcFrameworkCopySource = Path.Combine(PackagePath, "iOS", EmbraceXCFramework);
-            string xcFrameworkCopyDestination = Path.Combine(pathToBuiltProject, EmbraceXCFramework);
+            // Embed iOS frameworks
             
-            if (Directory.Exists(xcFrameworkCopyDestination))
-            {
-                var fileGuid = project.FindFileGuidByRealPath(xcFrameworkCopyDestination);
-                if (!string.IsNullOrWhiteSpace(fileGuid))
-                {
-                    project.RemoveFileFromBuild(project.GetUnityFrameworkTargetGuid(), fileGuid);
-                    project.RemoveFile(fileGuid);
-                }
-                Directory.Delete(xcFrameworkCopyDestination, true);
-            }
-
-            AssetDatabaseUtil.CopyDirectory(
-                xcFrameworkCopySource,
-                xcFrameworkCopyDestination,
-                true,
-                false);
-
-            #if UNITY_2021_3_OR_NEWER
-            // Later versions of Unity appear to automatically add the files to the linker phase.
-            // However there seems to be an issue with the copy process. To deal with this, we
-            // choose to overwrite the framework in position.
-            var xcFrameworkGuid = project.AddFile(
-                EmbraceXCFramework,
-                "Frameworks/io.embrace.sdk/iOS/Embrace.xcframework",
-                PBXSourceTree.Source);
+            /*
+             * It is worth noting that Unity *has* changed the default behavior of how xcframeworks are handled.
```

Comment:
@alyssa-embrace do we know for what versions of unity this change happened? not a big deal if we don't

Reply:
The changes seem to have occurred in updating some patch versions, which is a little frustrating from our end as it contributes to the fragility. Unfortunately, I don't currently know.